src/Layer/OGC3DTilesLayer.js

```diff
@@ -76,44 +129,80 @@ class OGC3DTilesLayer extends GeometryLayer {
 
         this.tilesRenderer.manager.addHandler(/\.gltf$/, itownsGLTFLoader);
 
-        // Set cache, download and parse queues to be shared amongst 3D tiles layers (waiting for 3d-tiles-renderer-js api change)
-        // this.tilesRenderer.lruCache = lruCache;
-        // this.tilesRenderer.downloadQueue = downloadQueue;
-        // this.tilesRenderer.parseQueue = parseQueue;
+        this._setupCacheAndQueues();
+        this._setupEvents();
 
         this.object3d.add(this.tilesRenderer.group);
 
-        // Add an initialization step that is resolved when the root tileset is loaded (see this._setup below)
+        // Add an initialization step that is resolved when the root tileset is loaded (see this._setup below), meaning
+        // that the layer will be marked ready when the tileset has been loaded.
         this._res = this.addInitializationStep();
     }
 
-    // TODO: what happens if the layer is added to multiple views? Should we store multiple tilesRenderer?
-    // How does it work for other layer types?
+    // Store points material config so they can be used later to substitute points tiles material by our own PointsMaterial
+    // These properties should eventually be managed through the Style API (see https://github.com/iTowns/itowns/issues/2336)
+    _handlePointsMaterialConfig(config) {
+        this.pntsMode = config.pntsMode;
+        this.pntsShape = config.pntsShape;
+        this.classification = config.classification;
+        this.pntsSizeMode = config.pntsSizeMode;
+        this.pntsMinAttenuatedSize = config.pntsMinAttenuatedSize || 3;
+        this.pntsMaxAttenuatedSize = config.pntsMaxAttenuatedSize || 10;
+    }
+
+    // Sets the lruCache and download and parse queues so they are shared amongst all tilesets
+    _setupCacheAndQueues() {
+        if (lruCache === null) {
+            lruCache = this.tilesRenderer.lruCache;
+        } else {
+            this.tilesRenderer.lruCache = lruCache;
+        }
+        if (downloadQueue === null) {
+            downloadQueue = this.tilesRenderer.downloadQueue;
+        } else {
+            this.tilesRenderer.downloadQueue = downloadQueue;
+        }
+        if (parseQueue === null) {
+            parseQueue = this.tilesRenderer.parseQueue;
+        } else {
+            this.tilesRenderer.parseQueue = parseQueue;
+        }
+    }
```

Comment:
Just a suggestion. You could add `lruCache`, `downloadQueue` and `parseQueue` in a single object and only do a single `null` check?